annotate.el

```diff
@@ -82,11 +82,21 @@
   "Face for annotation highlights."
   :group 'annotate)
 
+(defface annotate-highlight-secondary
+  '((t (:underline "turquoise")))
+  "Face for secondary annotation highlights."
+  :group 'annotate)
+
 (defface annotate-annotation
   '((t (:background "coral" :foreground "black")))
   "Face for annotations."
   :group 'annotate)
 
+(defface annotate-annotation-secondary
+  '((t (:background "turquoise" :foreground "black")))
```

Comment:
This is a very stark contrast. I would go with something more in the orange-yellow spectrum, akin to the alternating line highlights in tables.

Reply:
I chose the (more or less) complementary color, but without a valid reason actually! :)
I changed it to khaki, look nice to me anyway if you do not like it just tell me which one you prefer and i will add it. :)